client/src/components/common/Header/index.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { MenuIcon, headerLogo, MenuLogo } from '../../assets';
+import menuData from './menuData';
+import Button from '../Button';
+import './style.css';
+
+class Header extends Component {
+  state = {
+    isClosed: true,
+  };
+
+  onclick = () => {
+    const { isClosed } = this.state;
+    this.setState({ isClosed: !isClosed });
```

Comment:
rather than use `const { isClosed } = this.state;`
you can use this
```suggestion
    this.setState(prevState => ({ isClosed: !prevState.isClosed }));
```

Reply:
Done